Gemfile

```diff
@@ -58,9 +58,10 @@ group :development, :test do
   gem "factory_bot_rails"
 
   gem "brakeman", require: false
-  gem "rubocop", "0.93.1", require: false
-  gem "rubocop-rails", "2.9.1", require: false
-  gem "rubocop-performance", "1.10.2", require: false
+  gem "rubocop", require: false
```

Comment:
Any reason to unlock those at all?

Reply:
so that we don't get stuck on a specific version and we follow the rubocop versions (e.g. not getting too much behind) ?

Reply:
if there is a new cop we don't want, we can disable it, or usually autocorrect it

Reply:
Then please add at least some constraints if possible, not locked to individual versions, but opened using `~` on minor changes for example.

Reply:
I don't really agree, but sure I added some constraints on minor versions